contracts/wrapper/NameWrapper.sol

```diff
@@ -856,6 +937,11 @@ contract NameWrapper is
                 revert OperationProhibited(bytes32(id));
             }
         }
+
+        // delete token approval if CANNOT_APPROVE has not been burnt
+        if (fuses & CANNOT_APPROVE == 0) {
```

Comment:
Can we do this somewhere other than in a check?

Reply:
I'm not sure we can put it anywhere else, since safeBatchTransferFrom doesnt use `_transfer` (which is the only other place I thought we could put it, but safeBatchTransferFrom would also need to change this fuse. We could rename so it doesn't use the word check?

Reply:
That would at least be an improvement.